cmd/restore.go

```diff
@@ -110,8 +112,13 @@ func runRestore(flagSet *flag.FlagSet, cmdName, dbName, tableName string) error
 		}
 	case len(dbName) != 0 && len(tableName) == 0:
 		// database restore
+		fmt.Fprintf(os.Stderr, "dbName: %s\n", dbName)
```

Comment:
This looks like it was supposed to be removed

Reply:
+1